consensus/src/main/java/org/apache/iotdb/consensus/IConsensus.java

```diff
@@ -106,6 +107,17 @@ public interface IConsensus {
    */
   ConsensusGenericResponse changePeer(ConsensusGroupId groupId, List<Peer> newPeers);
 
+  /** Call this method on any member of originalGroup */
+  default ConsensusGenericResponse addNewNodeToExistedGroup(
```

Comment:
Add more comments like `createPeer` and `addPeer`?

Reply:
I've added the javadoc. Please take a look again, thanks!